src/blocks/Team/Team.tsx

```diff
@@ -0,0 +1,24 @@
+import { BiographyBlock } from '@/blocks/Biography/Biography'
+import type { Biography, TeamBlock as TeamBlockProps } from 'src/payload-types'
+
+type Props = TeamBlockProps
+
+export const TeamBlock = ({ team }: Props) => {
+  if (typeof team !== 'object') {
+    // TODO: figure out how to handle this - do we have access to the Payload client?
+    return <></>
+  }
+
+  const teamMembers: Biography[] = team.members.filter((member) => typeof member === 'object')
+
+  return (
+    <div className="container mx-auto py-12 px-4">
+      <h2 className="text-3xl font-bold text-center mb-12">{team.name}</h2>
+      <div className="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 gap-6">
+        {teamMembers.map((member, index) => (
+          <BiographyBlock key={index} biography={member} blockType={'biography'} />
```

Comment:
What was the reason for changing the member prop here? I think later on `biography.biography` is confusing. Maybe consider leaving it as `member`?

Reply:
The prop types for `BiographyBlock` are auto-generated by Payload from the `Block` configuration. Changing to `member` there would mean that the field is called `member` to the users in the admin panel - that feels a bit awkward as well? Not sure what you'd prefer here.

Reply:
I don't have a better nomenclature for this, so let's leave it.